packages/now-build-utils/package.json

```diff
@@ -21,6 +21,7 @@
     "@types/async-retry": "^1.2.1",
     "@types/cross-spawn": "6.0.0",
     "@types/end-of-stream": "^1.4.0",
+    "@types/execa": "2.0.0",
```

Comment:
You don't need `@types/execa` anymore.

> execa provides its own type definitions, so you don't need @types/execa installed!

[Source](https://www.npmjs.com/package/@types/execa)